### ISSUE TITLE
Adding Images to Guides

### ISSUE THREAD

**Issue by PardusEidolon** [PardusEidolon/Halo-3-Editing-Kit-Cookbook#11]
## Add images to guides

Go through the process of doing the guides and take snapshots of the process. If you want to take it a step further you can use [ScreenToGif](https://www.screentogif.com/) for your images instead.

## Images to Guide Tasks
- [ ] Blender Images to the H3 level guide
   - [ ] Tool Images/gifs to the H3 level guide
   - [ ] Guerilla Images/gifs to the H3 level guide
   - [ ] Sapien Images/gifs to the H3 level guide
 - [ ] Blender Images/gifs to the Custom weapon models guide
   - [ ] Tool Images/gifs to the Custom weapon models guide
   - [ ] Guerilla Images/gifs to the Custom weapon models guide
   - [ ] Sapien Images/gifs to the Custom weapon models guide

**Comment by A267-dev** [comment]
## Images to Guide Tasks for A267
- [ ] Decorator Brush Images/Gifs to the H3 level guide
   - [ ] Tool Images/gifs to the H3 level guide
   - [ ] Guerilla Images/gifs to the H3 level guide
   - [ ] Sapien Images/gifs to the H3 level guide

**Comment by PardusEidolon** [comment]
@A267-dev I added your task to the main thread.

**Comment by A267-dev** [comment]
> @A267-dev I added your tasks to the main thread.

Thank you Kindly, Though I can Merge them manually and save time in the future with Write Access

**Comment by A267-dev** [comment]
Nice work so far! Sorry I didn't get a chance to update my portions over the weekend, classes have me busy until Christmas.

**Comment by PardusEidolon** [comment]
Yeah, I was going to add a little more, I still need to figure out when the right time to use sharp and edge split, I have yet still to do the images for guerilla and sapien. But halo infinite is out so it might be some time...
